app/boards/shields/redox/Kconfig.defconfig

```diff
@@ -0,0 +1,53 @@
+
+if SHIELD_REDOX_LEFT
+
+config ZMK_KEYBOARD_NAME
+	default "Redox"
+
+config ZMK_SPLIT_BLE_ROLE_CENTRAL
+	default y
+
+endif
+
+if SHIELD_REDOX_LEFT || SHIELD_REDOX_RIGHT
+
+config ZMK_SPLIT
+	default y
+
+if ZMK_DISPLAY
+
+config I2C
+	default y
+
+config SSD1306
+	default y
+
+config SSD1306_REVERSE_MODE
+	default y
+
+endif # ZMK_DISPLAY
+
+if LVGL
+
+config LVGL_HOR_RES_MAX
+	default 128
+
+config LVGL_VER_RES_MAX
+	default 32
+
+config LVGL_VDB_SIZE
+	default 64
+
+config LVGL_DPI
+	default 148
+
+config LVGL_BITS_PER_PIXEL
+	default 1
+
+choice LVGL_COLOR_DEPTH
+	default LVGL_COLOR_DEPTH_1
+endchoice
+
+endif # LVGL
```

Comment:
I don't think this is needed since the Redox doesn't have displays from what I can tell.

Reply:
that is correct, removed